eland/ml/pytorch/pytorch_model.py

```diff
@@ -0,0 +1,154 @@
+#  Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Also can we make this module private like `_pytorch_model.py` so users know not to import things from here?